testConnectivity.py

```diff
@@ -477,7 +477,10 @@ def testConnectivity(args):
             dict_vfat3CalInfo[ohN] = getVFAT3CalInfo(dict_chipIDs[ohN],debug=args.debug)
             if args.debug:
                 print("dict_vfat3CalInfo[{0}]:\n{1}".format(ohN,dict_vfat3CalInfo[ohN]))
-            
+
+            #VFATs found in the database get biased later. This ensures all VFATs get biased.
+            vfatBoard.biasAllVFATs(dict_vfatMask[ohN])    
```

Comment:
The comment here is a bit vague as to the true nature of this line.  The goal here is to allow the user to have setup the VFAT configuration files to apply an IREF to *each* VFAT; then if a bit flip in the chipID results in a bad DB query the correct IREF value will still be applied.  Please change the comment to better reflect this.

Also squash your git commit history following directive from @jsturdy:

- [short instructions](https://github.com/cms-gem-daq-project/cmsgemos/pull/263#pullrequestreview-219547518)
- [detailed idea](https://github.com/cms-gem-daq-project/cmsgemos/pull/263#issuecomment-477534539)

An example of how to do this can be see [here](https://dev.to/blakedeboer/beginners-guide-to-interactive-rebasing-1ob)

Reply:
done

Reply:
In the future for a long comment like this consider using block comments, e.g.

```python
# this is a single line comment
"""
This is a block comment
It can be multi line
"""
```